tsfresh/utilities/dataframe_functions.py

```diff
@@ -372,7 +375,55 @@ def _normalize_input_to_internal_representation(timeseries_container, column_id,
     return timeseries_container, column_id, column_kind, column_value
 
 
-def roll_time_series(df_or_dict, column_id, column_sort, column_kind, rolling_direction, max_timeshift=None):
+def _roll_out_time_series(time_shift, grouped_data, rolling_direction, max_timeshift, min_timeshift,
```

Comment:
we should align the spelling of the arguments, e.g. time_shift versus max_timeshift, maybe remove the `time` from those argument names?

Reply:
I am ok with aligning, but I would want to keep the "time". As you have said, rolling barely makes sense in non-time domains.